### DISCUSSION TITLE
Chat/communication

### DISCUSSION THREAD

**Discussion by jinek** [Consolonia/Consolonia#144] [General]
Should we have a group, like, in Discord or Slack or anything else? For fast convenient way for communicate/synchronize. Or should we keep discussions fully here for visibility and future investigations? Or may be mix. Just checking if something could work better.

**Comment by tomlm** [comment]
I wouldn't mind having a real-time 1:1 communication mechanism.  I think discussions here is good, but if we want to have brainstorm/socialize sort of thing real-time is better.

I would propose for now to use this discussion for "open" discussions and perhaps we can use Teams for 1:1 for now?

**Reply by tomlm** [reply]
https://teams.live.com/l/invite/FBA7ROHYJdmgUooLQs ?

**Reply by jinek** [reply]
Can try, but teams is not the option for linux, need to think how to do it better

**Reply by tomlm** [reply]
Teams has both web app and desktop client 

https://woshub.com/install-microsoft-teams-linux/

**Reply by tomlm** [reply]
Webapp is full pwa

**Reply by jinek** [reply]
That's interesting. By the link instruction does not work. teams is not found there. Also official page proposes only PWA
![image](https://github.com/user-attachments/assets/60ead608-897c-4e65-a628-1267ebea7522)
Was it removed or something..
